content/articles/why-data-storage-is-shifting-to-the-blockchain/index.md

```diff
@@ -0,0 +1,60 @@
+
+This article discusses blockchain as a more secure, cost-friendly alternative data storage system. It also gets leaders prepared of what to expect of blockchain as a data storage solution.
+
+### Blockchain storage vs. cloud storage
+When assessing cloud storage and comparing it to the blockchain, you realize immediate differences between the two systems. For a start, the cloud is a virtual space where you can access data remotely. You gain access to the cloud by surfing on the internet.
+
+Blockchain storage is a little dissimilar because it is an encrypted system that uses different methodologies to encrypt and store data in protected databases. The blockchain is built in such a way that the system keeps the data across several nodes that form a consensus on the exact position of the data.
+
+This storage method is different from the cloud, where data remains in one location and is easily accessible to an individual as a whole. The way blockchain stores its data in small chunks distributed throughout the network adds a security layer. It ensures that if an individual or entity breaks into the system, they can only access a fraction of the data rather than the entire file.
+
+In cloud storage, data is mutable, while in blockchain, data is immutable. The cloud storage system is set up to let you change the already stored data. These modifications affect how another user perceives the data after the changes. Users cannot adjust the already existing data in the blockchain because the system bars them. You cannot delete or modify entries already made, which helps secure data from tampering.
+
+Cloud storage relies on third parties to ensure data security. It provides services using the Software as a Service, Platform as a Service, and Infrastructure as a Service models. Blockchain chooses to guarantee data protection and safeguard its integrity without involving third parties.
+
+Besides, data in the cloud can exist as either public or private. However, in blockchain, data is always public. Blockchain embraces transparency, meaning that an entity cannot hide data from other users. It enables individuals on the blockchain to track how data moves from one point to another.
+
+### Costs implications of storing data on the blockchain
+Unsurprising, such a storage technology that threatens to 'overthrow' cloud storage and offer a highly secure system has its share of financial challenges. In the last few years, the costs for implementing and running a blockchain system have [risen](https://content-blockchain.org/newsarchive/2017/07/20/exploding-costs-of-storing-data-on-a-blockchain/) tremendously. One reason for this is the increasing acceptance of blockchain technology has also led to a rise in data transmission and storage. Besides, the high demand for blockchain services has also driven up cryptocurrency prices.
```

Comment:
This paragraph sounds a bit off. Consider re-writing and splitting into two to convert a single idea per paragraph.